Felienne/hedy
# [QoL] Adds --watch mode to Tailwind script

**Description**
In this PR we make a minor QoL fix, adding `--watch` to the Tailwind script. Enabling us to run the Tailwind script in watch-mode and automatically generate a new .css file on each relevant file change. We already supported this for Typescript so it is a nice addition to create support for Tailwind as well.

**Fixes**
This PR fixes #2839.

**How to test**
Verify that the script still workings by simply running `./build-tools/heroku/tailwind/generate-css`. Then run the same script but add the `--watch` argument. Verify that the script stays "stuck" and only updates on every relevant file change (template, ts, styles.css).
